features/fixtures.py

```diff
@@ -0,0 +1,300 @@
+from behave import fixture
+from behave.fixture import use_composite_fixture_with, fixture_call_params
+from test_utils import json_helpers
+from test_utils.migrations import HasuraClient
+from utils.graphql_client import GraphQLClient
+from utils.stellar_client import StellarClient
+
+import settings
+
+import os.path
+import urllib.parse
+
+
+@fixture
+def graphql_client(context):
+    hasura_endpoint = context.config.userdata.get('hasura_endpoint')
+    graphql_endpoint = context.config.userdata.get('graphql_endpoint')
+    endpoint = urllib.parse.urljoin(hasura_endpoint, graphql_endpoint)
+    client = GraphQLClient(endpoint)
+    context.graphql_client = client
+    return client
+
+
+@fixture
+def database_seed(context):
+    hasura_endpoint = context.config.userdata.get('hasura_endpoint')
+    client = HasuraClient(hasura_endpoint)
+    shopozor_project = context.config.userdata.get('shopozor_project')
+    client.apply_migrations(shopozor_project)
+    fixtures_project = context.config.userdata.get('fixtures_project')
+    client.apply_migrations(fixtures_project)
+    yield client
+    client.rollback_migrations(fixtures_project)
+    client.rollback_migrations(shopozor_project)
+
+
+@fixture
+def stellar_snapshot(context):
+    client = StellarClient('shopozor-features')
+    client.create_snapshot()
+    has_worked = client.list_snapshots()
+    assert has_worked is True
```

Comment:
This is the little hack that lets the system wait for database snapshot completion. Without that hack, you would have random failures because sometimes the system needs background processes to finish the work, sometimes not.

Reply:
OK, I can't make informed feedback on that. You are on your own regarding acceptance tests and `behave`